src/index.html

```diff
@@ -11,12 +11,96 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="moyo">
+    <header class="moyo__header">
+      <a
+        href="#"
+        class="moyo__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+          class="moyo__icon"
```

Comment:
Fix classes everywhere
```suggestion
    <header class="header">
      <a
        href="#"
        class="header__link"
      >
        <img
          src="./images/logo.png"
          alt="moyo-logo"
          class="header__logo"
```